eduNEXT/platform-plugin-turnitin
# feat: add `es_419` and `es_ES` translations

### Description
This PR adds translation support to the plugin. Currently, it only adds the translation to **Spanish (Latin America)** and **Spanish (Spain)**.

### How to Test
1. Install this plugin with the changes in this branch in your environment.
2. Install `edx-ora2`, `openedx-events`, and `openedx-filters` with the latest versions. 
3. Change the platform language in (**Account** > **Site Preferences** > **Site Language**). Set it to **Spanish (Latin America)** or **Spanish (Spain)**.
4. Enabled the functionality by adding the following setting in your LMS: `ENABLE_TURNITIN_SUBMISSION = True`
5. In a unit of the course create a new ora assessment.
6. In the LMS you can see the Turnitin warning message translated into your selected language.

### Screenshots
![image](https://github.com/eduNEXT/platform-plugin-turnitin/assets/64033729/1c325dc6-9141-4510-b8a0-712985fdfc00)
